### PR TITLE
Replace BBC domains with wild cards and add new domain lists file

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #NUMBER

**Overall change:**
Tidies up our CSP utility by using wildcards for bbc domains thus reducing the number of hardcoded bbc domains and moves bbc domains and ad service country domains to a separate file for easier importing into the csp utility and its tests also improving readability of the main csp utility

**Code changes:**
- Remove all bbc domains and replace wiht a list of wildcard bbc domains
- Create a bbcDomains array and spread that into each CSP directive
- Move all adService country domains into a single array and spread those into the required directive
- Import and use bbc domains and adService domains list for use in the tests

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [X] I have assigned this PR to the Simorgh project

**Testing:**

I'd recommend a regression tests to ensure common things that CSP blocked will still work i.e. IDT includes, Adverts Fonts etc. Please ensure that we do not see any unexpected console errors for CSP violations.

You'd need to ensure the CSP headers are returned when using `npm run dev` as they are not usually returned on local development

- [X] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [X] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [X] This PR requires manual testing

### CODE REVIEW

File: src/server/utilities/cspHeader/index.js [hunk 1]
```diff
@@ -70,81 +68,14 @@ const advertisingDirectives = {
...
-    'https://adservice.google.co.kr',
-    'https://adservice.google.pl',
-    'https://adservice.google.com.mm',
-    'https://adservice.google.so',
-    'https://adservice.google.co.tz',
-    'https://adservice.google.com.np',
-    'https://adservice.google.lk',
-    'https://adservice.google.lv',
-    'https://adservice.google.ge',
+    ...adServiceCountryDomains,
```
Comment (simonsinclair) [inline src/server/utilities/cspHeader/index.js]: A minor suggestion for naming – should we replace `ad` with `advertising`? This is might be more identifiable. We also use `advertisingDirectives` on L12.

**Review by simonsinclair — APPROVED** [review]
Tidy!

File: src/server/utilities/cspHeader/index.js (hunk 1 above)
Comment (chris-hinds) [reply src/server/utilities/cspHeader/index.js]: Thats a good shout. Will do that now.

**Review by simonsinclair — APPROVED** [review]

**Review by pharingee — APPROVED** [review]

**Review by andrewscfc — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Checked few pages as part of regression and LGTM..

**Pull request merged**